### DISCUSSION TITLE
OCC commands doesn't work

### DISCUSSION THREAD

**Discussion by sebastiencastermans** [nextcloud/all-in-one#3346] [Questions] [answered]
### Steps to reproduce
1. Open a SSH session or go into the internal terminal
2. Tap an OCC command
3. That's all

### Expected behavior 
The OCC command run correctly.
### Actual behavior 
The OCC command dosen't run correctly.

### Host OS 
Linux - Ubuntu 22.04.3 LTS (GNU/Linux 6.2.0-32-generic x86_64)

#### Nextcloud AIO version 
v7.1.1
#### Current channel 
Stable
## Other valuable info 
I opened an issue here with the code that I use (the others OCC commands doesn't work too) and with the code that the terminal return to me but it was not a problem of that app but it's general. Therefore, others details are here : https://github.com/eldertek/duplicatefinder/issues/9#issue-1892234781

**Comment by Zoey2936** [comment, marked as answer]
https://github.com/nextcloud/all-in-one#how-to-run-occ-commands did you did it this way?

**Reply by sebastiencastermans** [reply]
Yes, sorry for the late 😬... It looks like the occ has change since the last time I use it. I see that the "-it " in my command has disappear now with the new version I suppose. Thanks for your link, it works 🙃.

**Comment by szaimen** [comment]
Hi, it looks like the groupfolders app causes this. IIRC there should be an update available in the app store that fixes this.

**Reply by sebastiencastermans** [reply]
Hi szaimen, with the link of Zoey2936 I found the solution and I don't search for the problem with the groupfolders app. Thank you for your response 😊.
